examples/npb-3.4.3/kit/npb.py

```diff
@@ -58,6 +71,7 @@ def bench_src_path(self) -> str:
     @staticmethod
     def get_build_var_names() -> List[str]:
         return [
+            "category",  # either "MPI" or "OMP"
```

Comment:
Have you tested the use of the MPI version?

I quickly tried it for the NPB-MZ at one point and it did not seem so trivial to get it to work

Reply:
no. This will have to be for later :)

Reply:
Okay, no problem. Resolving this conversation